integration/src/test/resources/raise-error-config.xml

```diff
@@ -121,4 +121,21 @@
         </error-handler>
     </flow>
 
+    <flow name="tryAndForEach">
+        <foreach collection="#[1 to 100]">
+            <try>
+                <test:processor/>
+                <raise-error type="APP:MY_TYPE"/>
+                <error-handler>
+                    <on-error-propagate>
+                        <logger message="Hello"/>
+                    </on-error-propagate>
+                </error-handler>
+            </try>
+        </foreach>
+        <error-handler>
+            <on-error-continue type="MULE:ANY"></on-error-continue>
```

Comment:
Shouldn't we validate explicitly if the continue is handling the error? If we don't and the raise error stops working this would show up as OK.